docs/src/gui/qtvcp-vismach.adoc

```diff
@@ -1,48 +1,57 @@
 :lang: en
 :toc:
 
-[[cha:qtvcp-vismach]]
-= QT Vismach
+[[cha:qtvcp:vismach]]
+= QtVismach
 
-Vismach is a set of Python functions that can be used to create and animate
-models of machines. +
-Vismach displays the model in a 3D viewport and the model
-parts are animated +
-as the values of associated HAL pins change.
+*Vismach* is a set of *Python functions that can be used to create and
```

Comment:
This text could have a section heading as well like you did in qtvcp.adoc.

Reply:
You mean `[[...]]` anchors ?

Reply:
Assumed yes and added title anchors.

Reply:
No, a heading as I said.
```suggestion
== Overview / Intro...
*Vismach* is a set of Python functions that can be used to create and
```

Reply:
OK, done.